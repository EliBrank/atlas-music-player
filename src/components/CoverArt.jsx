const CoverArt = () => {
  return (
    <>
      <img
        src="/images/cover-art.png"
        alt="placeholder cover art"
      />
    </>
  );
}

export default CoverArt;
