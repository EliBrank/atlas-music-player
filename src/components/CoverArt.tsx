const CoverArt = () => {
  return (
    <div className="flex justify-center">
      <img
        src="/images/cover-art.png"
        alt="placeholder cover art"
        width="400"
        className="mb-6"
      />
    </div>
  );
}

export default CoverArt;
