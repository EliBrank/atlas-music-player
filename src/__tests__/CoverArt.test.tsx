import { render } from "@testing-library/react";
import { expect, test } from "vitest";
import CoverArt from "@/components/CoverArt";

test("CoverArt renders correctly", () => {
  const { container } = render(<CoverArt coverArtURL="/placeholder.svg" />);
  expect(container).toMatchSnapshot();
});
