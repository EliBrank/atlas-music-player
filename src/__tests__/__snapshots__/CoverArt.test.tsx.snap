// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`CoverArt renders correctly 1`] = `
<div>
  <div
    class="flex justify-center"
  >
    <img
      alt="cover art"
      class="mb-6"
      src="/placeholder.svg"
      width="400"
    />
  </div>
</div>
`;
